### PR TITLE
Remove js error in all pages without pagination

### PR DESCRIPTION
**Pull request by femontanha**
Because inphinity.js when a page without pagination the js is broken and all js functions like Menu toggle stop working.
- I added if statement to test if the navSelector is defined
- Removed duplicated scrips on footer.html

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -9,7 +9,4 @@ <h2 class="newsletters-title">Mantenha-se atualizado!</h2>
       </form>
     </section>
   </div>
-</footer>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/zepto/1.1.6/zepto.min.js"></script>
```
Comment (fdaciuk) [inline _includes/footer.html:13]: I didn't understand this deletion.

Comment (femontanha) [inline _includes/footer.html:13]: I removed 3 scripts that are duplicated in _layouts/default.html

File: https://github.com/free-time/free-time.github.io/blob/master/_layouts/default.html

File: src/js/inphinity.js [hunk 2]
```diff
@@ -193,7 +193,10 @@ var inphinity = (function() {
             if (config.animationSpeed === "fast")
                 this.defaults.animationSpeed = 300;
         }
-        this.init();
+        
+        if(!!document.querySelector(config.navSelector)) {
```
Comment (fdaciuk) [inline src/js/inphinity.js]: -1
This code should be sent to [inphinity](https://github.com/raphamorim/inphinity)'s repository.

File: src/js/custom.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 // Start scroll infinite, popup and script echo.
-$(document).ready(function(){
+$(function(){
```
Comment (fdaciuk) [inline src/js/custom.js]: I think this update has already been done: https://github.com/femontanha/free-time.github.io/blob/master/src/js/custom.js

File: _includes/footer.html (hunk 1 above)
Comment (fdaciuk) [inline _includes/footer.html:13]: :+1:

**Comment by fdaciuk** [comment]
Thanks @femontanha! Please, just you come back the update on inphinity. For the rest, LGTM :+1: :)

**Comment by femontanha** [comment]
@fdaciuk I made a rollback for inphinity.js and create a pull request on base repository
https://github.com/raphamorim/inphinity/pull/12

**Comment by fdaciuk** [comment]
Very nice! Thank you :dancer: :dancer: :sparkles:

**Comment by raphamorim** [comment]
@femontanha thanks. Now it's merged :) 

Ref: https://github.com/raphamorim/inphinity/pull/12

**Pull request merged**